src/vsr.zig

```diff
@@ -1007,26 +1007,30 @@ pub fn quorums(replica_count: u8) struct {
     };
 }
 
-/// The SuperBlock's persisted VSR headers.
-/// One of the following:
-///
-/// - SV headers (consecutive chain)
-/// - DVC headers (disjoint chain)
-pub const ViewChangeHeaders = struct {
+pub const Headers = struct {
+    pub const Array = std.BoundedArray(Header, constants.pipeline_prepare_queue_max);
```

Comment:
This is pre-existing, but I don't feel great about this array -- it's sizeof is about a kilobyte, and, by using an array here, we are asking for memcpys.

Reply:
That is, I'd expect us to write to some exiting memory, rather than return it